### DISCUSSION TITLE
Unable to override exclusion of underscore-prefixed file

### DISCUSSION THREAD

**Discussion by sapphyrus** [retypeapp/retype#296] [General]
I've just upgraded my project to 2.0.0 and noticed that pages starting with a `_` are now missing from the generated site.

According to the retype documentation:
> By default, any file or folder name prefixed with a . or a _ will be excluded.
> 
> As well, any node_modules folder will be excluded.

Overriding this behavior worked just fine in `v1.11.2` with the following in my `retype.yml`:
```yml
include:
- _*
```

**Comment by geoffreymcgill** [comment]
okay, interesting. I'll run some tests. Your `include` configuration should still work and the functionality was not purposely changed.

Thanks for the report.

**Comment by geoffreymcgill** [comment]
Confirmed. It's a defect. Our goal is to fix right away and we will publish a public `v2.0.1` patch release as soon as the fix and testing are in place. 

Looks like this scenario was not covered by a unit test, so we'll get that in place too.

At the moment, I do not have a work-around available if you're using Retype `v2.0.0`. The only fix I know (at the moment) is to [uninstall](https://retype.com/guides/getting-started/#uninstall) `v2.0.0` and then [re-install](https://retype.com/guides/getting-started/#install) `v1.11.2`.

Thanks again for reporting the defect. I apologize for the inconvenience.

**Comment by sapphyrus** [comment]
Thanks for checking it out so quickly! I've downgraded to `v1.11.2` for now and I'm looking forward to the release of `v2.0.1`.

**Comment by geoffreymcgill** [comment]
We have this issue fixed and the fix will be included in the next public release of Retype, which will be `2.0.1`. Unless it is super critical for your project, we would prefer to wait a couple days before making the release. This will give us time to fix another bug we found today and see if any other important issues pop up.

**Reply by geoffreymcgill** [reply]
@sapphyrus The fix for this issue was included in the `v2.1.0` release and now publicly available.

You should be able to update to the latest release by using one of the following update commands:

https://retype.com/guides/getting-started/#update

Hope this helps.

**Reply by sapphyrus** [reply]
Thanks for the heads up, I've now upgraded to `v2.1.0` and can confirm it's working again.
